src/main/java/org/mockbukkit/mockbukkit/inventory/InventoryViewMock.java

```diff
@@ -145,15 +146,59 @@ public void setCursor(@Nullable ItemStack item)
 	@Override
 	public void setItem(int slot, @Nullable ItemStack item)
```

Comment:
Item needs to be copied before setting it in the inventory to retain paper parity